eva/third_party/huggingface/create.py

```diff
@@ -21,13 +21,34 @@
 from eva.catalog.catalog_type import ColumnType, NdArrayType
 from eva.catalog.models.udf_io_catalog import UdfIOCatalogEntry
 from eva.catalog.models.udf_metadata_catalog import UdfMetadataCatalogEntry
-from eva.third_party.huggingface.model import HFInputTypes
+from eva.third_party.huggingface.model import (
+    ASRHFModel,
+    AudioHFModel,
+    HFInputTypes,
+    ImageHFModel,
+    TextHFModel,
+)
 
 """
 We currently support the following tasks from HuggingFace.
 Each task is mapped to the type of input it expects.
 """
-SUPPORTED_TASKS = {
+SUPPORTED_TASKS = [
```

Comment:
Feels like repetitive information. Any reason for breaking it into a List and a Dict? The keys are the same, can probably just change the name of the dictionary.

Reply:
I agree its repetitive, but this just looks cleaner and makes it more obvious when we do operations on the list/dict. Renaming the dictionary can be a fair compromise. Will do.